Makefile

```diff
@@ -98,3 +113,9 @@ postgres_backups_clean:
 # psql -h postgres -U postgres
 # DROP DATABASE itou;
 # CREATE DATABASE itou OWNER itou;
+
+# Quick way to drop your local dev db if you corrupted it
+# and/or if you want to rebuild it from scratch.
+postgres_drop_local_data:
+	docker container rm itou_postgres ; echo continue even if failed
+	docker volume rm itou_postgres_data
```

Comment:
La méthode pour supprimer ta DB locale était juste au dessus dans le commentaire avec pour avantage de ne supprimer ni le conteneur ni le volume et donc de gagner en rapidité :)

En passant j'aimerais bien garder le Makefile le plus petit possible, je n'aimerais pas me retrouver avec un Makefile obèse à la LBB.

Reply:
Fair enough! Will remove it.